runtime/bcverify/vrfyhelp.c

```diff
@@ -579,6 +579,19 @@ isClassCompatible(J9BytecodeVerificationData *verifyData, UDATA sourceClass, UDA
 	if (NULL != verifyData->vmStruct->currentException) {
 		return (IDATA) FALSE;
 	}
+	
+	if (J9ROMCLASS_IS_HIDDEN(verifyData->romClass)) {	
+		J9UTF8* className = J9ROMCLASS_CLASSNAME(verifyData->romClass);
+		if (sourceLength == J9UTF8_LENGTH(className)
+			&& (0 == strncmp(sourceName, J9UTF8_DATA(className), sourceLength))
+		) {
+			/* isRAMClassCompatible() cannot find ram class of a hidden class, we are testing if 
```

Comment:
Can a hidden class extend a hidden class?  I think not as it won't be findable to tell if the superclass has been loaded or not.  Assuming I'm right on that, can you add a comment to that effect here?

Reply:
I agree with you. I don't think a class can extend a hidden class.

Reply:
Comment added.